.npmrc

```diff
@@ -1 +1,2 @@
-shamefully-hoist=false
\ No newline at end of file
+public-hoist-pattern[]=*import-in-the-middle*
+public-hoist-pattern[]=*require-in-the-middle*
```

Comment:
The sentry-related errors in the console were driving me crazy so fixed it according to their docs https://docs.sentry.io/platforms/javascript/guides/node/troubleshooting/#pnpm-resolving-import-in-the-middle-external-package-errors